ansible_base/feature_flags/migrations/0001_initial.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.17 on 2025-01-22 22:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FeatureFlag',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+            ],
+        ),
+    ]
```

Comment:
Can we skip adding the Model and migration for now?

it looks like for this phase the model is not needed